src/components/Extensibility/ExtensibilityWidgets.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import { ExtensibilityErrBoundary } from 'components/Extensibility/ExtensibilityErrBoundary';
+import { useGetSchema } from 'hooks/useGetSchema';
+
+import { DataSourcesContextProvider } from './contexts/DataSources';
+import { useGetWidgets } from './useGetWidget';
+import { Widget } from './components-widget/Widget';
+import { TranslationBundleContext } from './helpers';
+import { useJsonata } from './hooks/useJsonata';
+import { usePrepareResourceUrl } from 'resources/helpers';
+import pluralize from 'pluralize';
+import { useGet } from 'shared/hooks/BackendAPI/useGet';
+
+export const ExtensibilityWidgetsCore = ({ resMetaData, root }) => {
```

Comment:
Shouldn't this be called `ExtensibilityWidgetCore`, since it's the core of a single `ExtensibilityWidget`, not `ExtensibilityWidgets`?